ConsoleApplication1/ConsoleApplication1/Positif_number_using_linq.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Linq;
+
+namespace ConsoleApplication1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] numbers = { 1, 2, -8, 10, 7, -6, 8 };
+            var result = GetpositifNumber(numbers);
+
+            foreach (var item in result)
+            {
+                Console.WriteLine(item);
+            }
+            Console.ReadKey();
+        }
+        static char[] GetpositifNumber(int[] array)
```

Comment:
Лучше использовать camelCase в именах переменных, методов, полей, свойств etc. Также английские слова лучше писать без ошибок.
```suggestion
        static char[] GetPositiveNumber(int[] array)
```